bitnami/discourse/README.md

```diff
@@ -36,9 +36,11 @@ Looking to use Discoursereg; in production? Try [VMware Application Catalog](htt
 To install the chart with the release name `my-release`:
 
 ```console
-helm install my-release oci://registry-1.docker.io/bitnamicharts/discourse
+helm install my-release oci://registry_name/repository_name/discourse
 ```
 
+> Please replace the registry_name and repository_name placeholders with a reference to your chart registry & repository.
```

Comment:
```suggestion
> Note: you need to substitute the placeholders REGISTRY_NAME and REPOSITORY_NAME with a reference to your Helm chart registry and repository.
```

According to other examples of placeholders, they are defined in uppercase and with a similar wording as the suggested one